packages/yoroi-extension/chrome/content-scripts/inject.js

```diff
@@ -116,6 +116,8 @@ function listenToBackgroundServiceWorker() {
                             }
                         }, location.origin);
                     }
+                } else {
+                    connected = true;
```

Comment:
I am not sure this is the correct place for this flag. Let me investigate this in greater details first. Marking to not be merged for now